keras_cv/models/object_detection/retina_net/retina_net.py

```diff
@@ -471,18 +472,17 @@ def train_step(self, data):
 
     def test_step(self, data):
         x, y = data
-        y_for_metrics, y_training_target = y
+        gt_boxes = y["gt_boxes"]
+        gt_classes = y["gt_classes"]
         y_pred = self(x, training=False)
-        _ = self._backward(y_training_target, y_pred)
+        _ = self._backward(gt_boxes, gt_classes, y_pred)
 
         # Early exit for no custom metrics
         if not self._includes_custom_metrics:
```

Comment:
can remove this block if we're never decoding predictions

Reply:
Done